requests_openapi_client/core.py

```diff
@@ -369,10 +369,10 @@ def _collect_operations(cls):
                     log.warning(
                         f"multiple '{operation_id}' found , operation ID should be unique"
                     )
-                    v = self._operations[operation_id]
+                    v = cls._operations[operation_id]
```

Comment:
You fell victim to one of the classic blunders. The most famous is "don't get involved in a land war in Asia", but only slightly less well known is this:

"Python no static analysis"